### DISCUSSION TITLE
Showcase: Collection of animal noises for RFID chips

### DISCUSSION THREAD

**Discussion by jnettels** [MiczFlor/RPi-Jukebox-RFID#2252] [Show and tell]
Hi everyone,

maybe this is useful to anyone else:
A (german) google search for the words "google tierlaute" yields a user interace with 57 animal noises. ChatGPT wrote a script for me that uses part of the html from that search result page to download both the mp3 sound files and the icon images.

The URLs look like this:
www.google.com/logos/fnbx/animal_sounds/dinosaur.mp3
www.google.com/logos/fnbx/animal_sounds/dinosaur.png

The files are arranged and renamed into folders, which can be copied to a phoniebox.

Also a DIN A4 paper sheet is created with the animal icons arranged to fit onto 2.5 cm round RFID chips, when printed onto sticker paper.
A circle is printed around each image to provide guidance for cutting the stickers.

I do not want to host the files myself, but you can run the script to get the files yourself and adapt it to your needs:
https://gist.github.com/jnettels/89e62a4dc6672d467d180529d04914bb

This is the printable DIN A4 page:
![a4_collage](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/8526498/d95ed44b-3e37-4793-9ac0-39bd54ddcb39)

And the result can look like this
![Animal noises stickers](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/8526498/e38572a2-de0b-4f5d-9ab5-a788149a6f8e)

Cheers, and thanks for this awesome project!

**Comment by pabera** [comment]
This is a really cool use case, also in combination with GPT. Thanks for sharing this.
